openapi.yaml

```diff
@@ -208,10 +497,378 @@ definitions:
+                $ref: '#/definitions/MaximumBet'
+              betPool:
+                $ref: '#/definitions/BetPool'
+              tableCards:
+                $ref: '#/definitions/Cards'
+            additionalProperties: false
+        required: 
+          - type
+          - id
+        additionalProperties: false
+    required:
+      - data
+    additionalProperties: false
+  GamePostBody:
+    properties:
+      data:
+        type: object
+        properties:
+          type:
+            $ref: '#/definitions/GameType'
+          attributes:
+            type: object
+            properties:
+              memberIds:
+                $ref: '#/definitions/MemberIds'
+              minimumBet:
+                $ref: '#/definitions/MinimumBet'
+              maximumBet:
+                $ref: '#/definitions/MaximumBet'
+            required:
+              - playerIds
+              - minimumBet
+              - maximumBet
+            additionalProperties: false
+        required: 
+          - type
+          - attributes
+        additionalProperties: false
+    required:
+      - data
+    additionalProperties: false
+  GameResource:
+    properties:
+      id:
+        $ref: '#/definitions/GameId'
+      type:
+        $ref: '#/definitions/GameType'
+      attributes:
+        type: object
+        properties:
+          round:
+            $ref: '#/definitions/Round'
+          playerIds:
+            $ref: '#/definitions/PlayerIds'
+          minimumBet:
+            $ref: '#/definitions/MinimumBet'
+          maximumBet:
+            $ref: '#/definitions/MaximumBet'
+          betPool:
+            $ref: '#/definitions/BetPool'
+          tableCards:
+            $ref: '#/definitions/Cards'
+      links:
+        $ref: '#/definitions/SelfLink'
+  GameResult:
+    properties:
+      links:
+        $ref: '#/definitions/SelfLink'
+      data:
+        $ref: '#/definitions/GameResource'
+  GameResults:
+    properties:
+      links:
+        $ref: '#/definitions/SelfLink'
+      data:
+        type: array
+        items:
+          $ref: "#/definitions/GameResource"
+  GameType:
+    type: string
+    enum:
+      - game
+  MaximumBet:
+    type: integer
+    minimum: 1
+    description: The maximum bet every player can place each round.
+    example: 2000
+  MinimumBet:
+    type: integer
+    minimum: 1
+    description: The minimum bet every player have to place at least each round.
+    example: 1000
+  PlayerBet:
+    type: integer
+    minimum: 0
+    description: The bet from a specific player.
+  MemberEmail:
+    type: string
+    example: abc@efg.com
+    format: email
+  MemberExpAboveLevel:
+    type: integer
+    minimum: 0
+    description: Player's experience point above current level.
+    example: 1234
+  MemberId:
+    type: string
+    example: '101'
+  MemberIds:
+    type: array
+    items:
+      $ref: '#/definitions/MemberId'
+  PlayerId:
+    type: string
+    example: 'g1m1'
+  PlayerIds:
```

Comment:
I think after you change playerIds to memberIds in the places I commented on, this definition won't be used anywhere and you should be able to delete it

Reply:
I didn't. The resource are requested and returned with playerId mostly on this endpoint.